server/lib/nl/config_builder/base.py

```diff
@@ -32,6 +32,24 @@
 from server.lib.nl.fulfillment.types import SV2Thing
 import server.lib.nl.fulfillment.utils as futils
 
+_SPECIAL_REPLACEMENTS = {
+    " A ": " a ",
```

Comment:
nit: if you're going to make another change I'd include `at` too.

Reply:
Done